site/docs/v1/tech/tutorials/integrate-laravel-api.md

```diff
@@ -0,0 +1,245 @@
+
+This will create the FusionAuth configuration for your {{page.technology}} API.
+
+```shell
+php setup.php "YOUR_API_KEY_FROM_ABOVE"
+```
+
+If you want, you can [log into your instance](http://localhost:9011) and examine the new Application the script created for you.
+
+## Create Your {{page.technology}} API
+
+Now you are going to create a {{page.technology}} application. While this section uses a simple {{page.technology}} application, you can use the same steps to integrate any {{page.technology}} application with FusionAuth.
+
+First, make a directory:
+
+```shell
+mkdir ../setup-laravel && cd ../setup-laravel
+```
+
+Next, create a simple {{page.technology}} template using Laravel's build script.
+
+```shell
+curl -s "https://laravel.build/fusionauth-example-laravel-api?with=mariadb" | bash
+```
+
+This can take several minutes to complete, so please be patient.
+
+### Adding JWT Authentication
+
+After it is done, change into the `fusionauth-example-laravel-api` directory and install [`tymon/jwt-auth`](https://github.com/tymondesigns/jwt-auth), a library that adds JWT capabilities to Laravel.
+
+```shell
+cd fusionauth-example-laravel-api
+./vendor/bin/sail composer require tymon/jwt-auth
+```
+
+Add some authentication routes to `routes/api.php`. 
+
+```diff
+{% remote_include https://github.com/FusionAuth/fusionauth-example-laravel-api/commit/49a4e7c3e3610403c6a00246bf5b997bc44b5bd4.diff %}
+```
+
+Create `app/Http/Controllers/AuthController.php` to hold the authentication process.
+
+```php
+{% remote_include https://raw.githubusercontent.com/FusionAuth/fusionauth-example-laravel-api/main/laravel/app/Http/Controllers/AuthController.php %}
+```
+
+Laravel uses something called [Guards](https://laravel.com/docs/10.x/authentication#adding-custom-guards) to protect your endpoints, so we need to tell it about the new Guard provided from that library.
+
+```diff
+{% remote_include https://github.com/FusionAuth/fusionauth-example-laravel-api/commit/5fb7134a70e1adee39f2d1d64ce4ed5ac7303a47.diff %}
+```
+
+To make the library available for use, publish its configuration by running the command below.
+
+```shell
+./vendor/bin/sail artisan vendor:publish --provider="Tymon\JWTAuth\Providers\LaravelServiceProvider"
+```
+
+### Editing Files
+
+The Laravel installer already brings some useful resources for many applications. We'll have to edit a few of them and add some more.
+
+First, remove the need for users to have a password by editing `app/Models/User.php`.
+
+```diff
+{% remote_include https://github.com/FusionAuth/fusionauth-example-laravel-api/commit/e4298a10437ff396fe3dfc02c3bcf85f310921a0.diff %}
+```
+
+Change the created migration at `database/migrations/2014_10_12_000000_create_users_table.php`.
```

Comment:
Really, that migration from 2014 is in the project?

Reply:
Yeah, it's from [the base template from Laravel](https://github.com/laravel/laravel/blob/10.x/database/migrations/2014_10_12_000000_create_users_table.php)